app/src/assets/scss/custom/ui/organism/product-category/product-category.scss

```diff
@@ -0,0 +1,16 @@
+.product-category-section {
```

Comment:
jeżeli komponent nazywa się ProductCategory to ta klasa jeżeli jest w pierwszym divie to najlepiej jakby sie nazywały es-product-category